platform/cc3200-launchxl/apps/blink.c

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Any reason to include this example in the platform folder?
Usually examples/[platform]/ is being used